outputs.tf

```
@@ -0,0 +1,4 @@
+output "security_group_id" {
+  value       = "${aws_security_group.default.id}"
```

```suggestion
  value       = "${join("", aws_security_group.default.*.id}"
```

since we have `count` on all resources (because of `enabled` var), all their outputs are lists.
better to use the `splat+join` pattern, other wise it would create some issues in top-level modules (we've seen that many times)